spec/features/formating/column_separator_spec.rb

```diff
@@ -148,6 +156,16 @@
         end.to raise_exception SmarterCSV::NoColSepDetected
       end
 
+      it 'does not detect separators that are between quotes' do
+        data = SmarterCSV.process(
+          "#{fixture_path}/separator_chars_between_quotes_no_headers.csv",
+          options.merge(user_provided_headers: %w[Name Age Job Department Project])
```

Comment:
```suggestion
          options.merge(headers_in_file: false, user_provided_headers: %w[Name Age Job Department Project])
```

Reply:
important to add `headers_in_file: false`, otherwise the first line will be ignored